app/params/genesis.go

```diff
@@ -41,6 +46,98 @@ type GenesisParams struct {
 	ClaimDurationOfDecay    time.Duration
 }
 
+func MainnetGenesisParams() GenesisParams {
+	genParams := GenesisParams{}
+
+	genParams.AirdropSupply = sdk.NewIntWithDecimal(1, 14) // 10^15 uosmo, 10^8 (100 million) osmo
+	genParams.ChainID = "osmosis-1"
+	// genParams.GenesisTime = time.Now() // TODO: Finalize date
+
+	genParams.NativeCoinMetadata = banktypes.Metadata{
+		Description: fmt.Sprintf("The native token of Osmosis"),
+		DenomUnits: []*banktypes.DenomUnit{
+			{
+				Denom:    BaseCoinUnit,
+				Exponent: 0,
+				Aliases:  nil,
+			},
+			{
+				Denom:    HumanCoinUnit,
+				Exponent: OsmoExponent,
+				Aliases:  nil,
+			},
+		},
+		Base:    BaseCoinUnit,
+		Display: HumanCoinUnit,
+	}
+
+	genParams.StakingParams = stakingtypes.DefaultParams()
+	genParams.StakingParams.UnbondingTime = time.Hour * 24 * 7 * 2 // 2 weeks
+	genParams.StakingParams.MaxValidators = 100
+	genParams.StakingParams.BondDenom = genParams.NativeCoinMetadata.Base
+	genParams.StakingParams.MinCommissionRate = sdk.MustNewDecFromStr("0.05")
+
+	genParams.MintParams = minttypes.DefaultParams()
+	genParams.MintParams.EpochIdentifier = "daily"                                      // 1 week
+	genParams.MintParams.GenesisEpochProvisions = sdk.NewDec(300_000_000).QuoInt64(365) // 300M / 365 = ~821917.8082191781
+	genParams.MintParams.MintDenom = genParams.NativeCoinMetadata.Base
+	genParams.MintParams.ReductionFactor = sdk.NewDec(2).QuoInt64(3) // 2/3
+	genParams.MintParams.ReductionPeriodInEpochs = 365               // 1 year (screw leap years)
+	genParams.MintParams.DistributionProportions = minttypes.DistributionProportions{
+		Staking:          sdk.MustNewDecFromStr("0.25"), // 25%
+		DeveloperRewards: sdk.MustNewDecFromStr("0.25"), // 25%
+		PoolIncentives:   sdk.MustNewDecFromStr("0.5"),  // 50%  TODO: Reduce to 45% once Community Pool Allocation exists
+	}
+	genParams.MintParams.MintingRewardsDistributionStartTime = genParams.GenesisTime.Add(24 * time.Hour) // TODO: Finalize
+	// genParams.MintParams.DeveloperRewardsReceiver
+
+	genParams.DistributionParams = distributiontypes.DefaultParams()
+	genParams.DistributionParams.BaseProposerReward = sdk.MustNewDecFromStr("0.01")
+	genParams.DistributionParams.BonusProposerReward = sdk.MustNewDecFromStr("0.04")
+	genParams.DistributionParams.CommunityTax = sdk.MustNewDecFromStr("0")
+	genParams.DistributionParams.WithdrawAddrEnabled = true
+
+	genParams.GovParams = govtypes.DefaultParams()
+	genParams.GovParams.DepositParams.MaxDepositPeriod = time.Hour * 24 * 14 // 2 weeks
+	genParams.GovParams.DepositParams.MinDeposit = sdk.NewCoins(sdk.NewCoin(
+		genParams.NativeCoinMetadata.Base,
+		genParams.AirdropSupply.QuoRaw(1_000_000), // 1 millionth of airdrop supply
+	))
+	genParams.GovParams.TallyParams.Quorum = sdk.MustNewDecFromStr("0.25") // 25%
```

Comment:
Make quorum 15%

Reply:
I'd really prefer to keep it at 25%, it already feels low but ~somewhat justifiable due to software issues + very quick voting periods at launch

Reply:
ok